### ISSUE TITLE
accounts_cstm not found in database error yaai v3

### ISSUE THREAD

**Issue by cmould** [blak3r/yaai#136]
I have been trying to install yaai 3 on SugarCrm 6.5.13 (after giving up on 2.6). I have reinstalled sugarcm so i have a fresh install. I noticed in the sugarcrm.log that I am getting a fatal error on the missing accounts_cstm. i grepped both the yaai 2.6 and yaai 3 source and find that this table is a new reference in yaai 3. i need the queue feature so I would like to get this working. My install seems to detect calls and properly posts the call info - it is visible in the UI. However i am not getting any popups. That led  me to look at the sugarcrm log where I found the errors. 

My external line is a siptrunk from my production PBX which is 1.2. I am installing a new SugarCRm and Asterisk PBX 1.8 - this is where i am testing. My user extension is 4 digits and my account extension is 7 digits. Any assistance is appreciated and more importantly when are we getting a release for v3. Does it work currently, is there an undocumented step to get accounts_cstm setup? I have looked at the account layout file after installing the module but i see no custom fields required for v3.

**Comment by blak3r** [comment]
Create a custom field in studio for accounts and contacts.

**Comment by cmould** [comment]
Thanks for the response - I did that, also considered commenting out the join - attributes from accounts_cstm are not used, however i created a test field but still no popups - i am also getting a handlbars undefined in Chrome console. Any fixes for that?

**Comment by blak3r** [comment]
What branch are you using?
I'm working on finalizing the release today.  I thought the newly merged
master branch worked.

**Comment by cmould** [comment]
Downloaded v3 about yaai-v3 June 19.

**Comment by blak3r** [comment]
Give the master branch a try... V3 got kinda messed up.  Long story.

**Comment by cmould** [comment]
Good morning Blake I tried v3 but was not successful I noted calllistener was not included in the download. Not sure if this is included elsewhere and being a newbie I chose not to pursue v3. I have opted to spend the time on 2.6b and was successful. You indicated a v3 release was coming, what is the ETA? I do need the queue feature  
Sent from my Verizon Wireless BlackBerry

-----Original Message-----
